statics/workflows/traceroute.yaml

```diff
@@ -0,0 +1,110 @@
+UUID: "9b598b28-b52e-488f-7b1c-402ac07e59b5"
+name: "Traceroute"
+description: "Traceroute"
+parameters:
+  - name: protocol
+    description: Protocol
+    type:  choice
+    default: icmp4
+    values:
+      - description: "Protocol : ICMPv4/Echo request"
+        value: icmp4
+      - description: "Protocol : TCP/IPv4"
+        value: tcp4
+  - name: source
+    description: Source Node
+    type: node
+  - name: destination
+    description: Destination Node
+    type: node
+source: |
+  function Traceroute(protocol, from, to) {
+    var sources = [];
+    var result = {};
+    var From = {};
```

Comment:
don't differentiate a variable just using the uppercase. it will confuse. use some other names

Reply:
Done, Thanks!